cypress/integration/01 - signup-login.ts

```diff
@@ -1,17 +1,19 @@
 describe("Signup/Login", () => {
+    const email = `${Math.floor(Math.random() * 1e8)}@example.com`;
```

Comment:
While debugging the tests I noticed that I couldn't easily replay the signup test once it had passed once, since the second time the account already existed. This made debugging really annoying because I had to rerun the `test:e2e:dev` command every time instead of just clicking the replay button. I thought about ways to reset the server state before every test run, but ultimately decided it was easier to simply use a randomised email.

Reply:
Nice one. I didn't do this initially because of the "hardocded email" need.